### DISCUSSION TITLE
A security issue in Github Repository

### DISCUSSION THREAD

**Discussion by abdollahkahne** [community/community#17890] [Other Feature Feedback, Questions, & Ideas] [answered]
I think I found a security issue in one of GitHub repository. How I should report it securely? Is there any mechanism defined for it?
Thanks

**Comment by ettaboyle** [comment, marked as answer]
Hi @abdollahkahne 👋🏻     

We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

* [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
* [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
* [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
* [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you,

The GitHub Community Team
